core/src/main/scala/com/mesosphere/usi/core/Scheduler.scala

```diff
@@ -1,13 +1,65 @@
 package com.mesosphere.usi.core
 
-import com.mesosphere.usi.core.models.PodSpec
+import akka.NotUsed
+import akka.stream.BidiShape
+import akka.stream.scaladsl.{BidiFlow, Broadcast, GraphDSL}
+import com.mesosphere.usi.models.{Mesos, SpecEvent, USIStateEvent}
 
-import scala.concurrent.Future
 
-class Scheduler {
+/*
+ * Provides the scheduler graph component. The component has two inputs, and two outputs:
+ *
+ * Input:
+ * 1) SpecEvents - Used to replicate the specification state from the framework to the USI scheduler
+ * 2) MesosEvents - Events from Mesos; offers, task status updates, etc.
+ *
+ * Output:
+ * 1) USIStateEvents - Used to replicate the state of pods, agents, and reservations to the framework
```

Comment:
Nit: Through out this review I kept reading `USIStateEvent` as `USIS-tate-events` where `USIS` sounds somewhat like `ISIS` 😱  I know USI is an abbreviation but could we rename it to `UsiStateEvent` or even just `SchedulerStateEVent`?

Reply:
Yeah, please not too many caps. I once worked on an `EMLVWProcessorImplFactory`. Why not simply `StateEvents`?

Reply:
Remnants of comments from before renames :(